### ISSUE TITLE
New players cannot claim any land

### ISSUE THREAD

**Issue by Koekie78** [SalvosGG/issue-tracker#21]
Maybe it's because we're still in alpha that I started without any money. To claim your first chunk you need 10k. In my opinion new players should be able to claim at least one chunk to keep their stuff safe from others.

This could be solved by:
- letting players start with 10k (fools will spend it before claiming)
- making the first chunk free/very cheap to claim (seems safer, but is it possible to configure?)

**Comment by Koekie78** [comment]
Might be a bit more difficult to fix then I thought. You need 10k to create a land (maybe new players don't want to and just want to join one) + first chunk costs 1K

**Comment by Morphiee** [comment]
On first join, players receive a campfire that is a base camp of sorts. It claims 1 chunk for 24 hours (Then it removes and they can claim it themselves.) We can alternatively just up the time the camp last and its radius by a tad? We can also take a look at the lands scaling when it comes to creating a land and claiming chunks.

**Comment by Koekie78** [comment]
Ah, the campfire was not clear to me. Upping duration seems like a good idea. That will help less active players. The radius I would keep at 1 chunk, otherwise players may build in the full area and not have earned enough when it expires to claim the chunks. Chunk cost itself seems reasonable so far (I only have 6). No real need to change that imo

**Comment by UnAlguien** [comment]
chunk cost def needs reworking, its a 71k jump from 17 to 18 chunks according to crad

**Comment by Morphiee** [comment]
Okay, SO I did a bunch of balancing in terms of claim costs and the campfire for starting out.

Campfires now are 3 chunks that last 5 days and each player gets one when first joining.

Lands:
The cost to Create a land is now  5,000 with the first chunk being free. ( cost to create was $10,000)

Your first chunk will cost $1,000 from there and scale 10% of the cost after that. For reference, it was around $2,000 to start and a 25% scale of cost per chunk so it is significantly lowered. chunks 44-45 cost me about 60k in my testing. For a 45 chunk claim, I spent around 700k (That is ALOT of land.) We should be good with this current scaling and can tweak it in the future if needed based on the economy.
